lib/src/clipboard/clipboard.dart

```diff
@@ -0,0 +1,16 @@
+
+import 'package:flutter/services.dart';
+import 'package:flushbar/flushbar.dart';
+
+final flushbarCopied = Flushbar<dynamic>(
```

Comment:
Try to avoid `dynamic` type as it doesn't give any information. So replace this with a real type (preferred option) or don't put it